matchbox_socket/src/ggrs_socket.rs

```diff
@@ -1,59 +1,34 @@
-use ggrs::{PlayerType, UdpMessage};
-use std::{
-    collections::{hash_map::DefaultHasher, HashMap},
-    hash::{Hash, Hasher},
-    net::{Ipv6Addr, SocketAddr},
-};
+use ggrs::{Message, PlayerType};
 
 use crate::{webrtc_socket::MessageLoopFuture, WebRtcSocket};
 
 #[derive(Debug)]
 pub struct WebRtcNonBlockingSocket {
     socket: WebRtcSocket,
-    fake_socket_addrs: HashMap<String, SocketAddr>,
-    fake_socket_addrs_reverse: HashMap<SocketAddr, String>,
 }
 
 impl WebRtcNonBlockingSocket {
     #[must_use]
     pub fn new<T: Into<String>>(room_url: T) -> (Self, MessageLoopFuture) {
         let (socket, message_loop) = WebRtcSocket::new(room_url);
-        (
-            Self {
-                socket,
-                fake_socket_addrs: Default::default(),
-                fake_socket_addrs_reverse: Default::default(),
-            },
-            message_loop,
-        )
+        (Self { socket }, message_loop)
```

Comment:
Now it kind of almost look like we could get away with just directly implementing the trait on WebRtcSocket itself.